flask_smorest/prefixed_config.py

```diff
@@ -0,0 +1,16 @@
+from collections.abc import Mapping
+
+
+class PrefixedConfigProxy(Mapping):
```

Comment:
Maybe I should move it to `utils.py`?

Reply:
Yes. And you cold make it more generic, like `PrefixedMappingProxy`, taking proxied `dict` as input rather than `app`.

Then you'd do

```py
self.config = PrefixedMappingProxy(app.config, self.config_prefix)
```